CMakeLists.txt

```diff
@@ -4,6 +4,13 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(SKIP_GOOGLETEST_INSTALL "Skip installation" ${SKIP_GOOGLETEST_INSTALL} )
```

Comment:
...which it has to because of the `option` anti-pattern being employed. I'm inclined to agree; `option(ENABLE_GOOGLETEST_INSTALL "Enable installation of Google Test. (Projects embedding Google Test may want to turn this OFF.)" ON)` would very possibly be better...

If you're *really* insistent on not using `option` the way it's meant to be used, this would be better:
```
if(NOT DEFINED ENABLE_GOOGLETEST_INSTALL)
  set(ENABLE_GOOGLETEST_INSTALL ON)
endif()
option(ENABLE_GOOGLETEST_INSTALL "..." ${ENABLE_GOOGLETEST_INSTALL})
```